tests/v2/validation/provisioning/ssh.go

```diff
@@ -0,0 +1,31 @@
+package provisioning
+
+// This file contains all tests that require to ssh into a node to run commands to check things such as any stats, benchmarks, etc. For example, ssh is required to check the cpu usage of a process running on an individual node.
+
+import (
+	"strconv"
+
+	"github.com/rancher/rancher/tests/framework/pkg/nodes"
+)
+
+// Constants will be stored here to be referenced throughout the tests written here
+const (
+	cpuUsageVar = 30
+)
+
+// This func checks the cpu usage of the cluster agent. If the usage is too high the func will return a warning.
+func CheckCPU(node *nodes.Node) (string, error) {
+	command := "ps -C agent -o %cpu --no-header"
+	// set the limit for cpu usage
```

Comment:
Same as above, you can remove this comment. The comment for the function gives the user an explanation as to what is going on here.